src/composables/useGetParticipants.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @copyright Copyright (c) 2024 Dorra Jaouad <dorra.jaoued1@gmail.com>
+ *
+ * @author Dorra Jaouad <dorra.jaoued1@gmail.com>
+ * @author Marco Ambrosini <marcoambrosini@icloud.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+import debounce from 'debounce'
+import { ref, onMounted, nextTick, computed } from 'vue'
+
+import { subscribe, unsubscribe } from '@nextcloud/event-bus'
+
+import { useIsInCall } from './useIsInCall.js'
+import { useStore } from './useStore.js'
+import { CONVERSATION } from '../constants.js'
+import { EventBus } from '../services/EventBus.js'
+
+/**
+ * @param {boolean} isActive whether the participants tab is active
+ */
+export function useGetParticipants(isActive = true) {
```

Comment:
I haven't checked if it affects current composable usage, but for a general case for composables development.

If a composable's param is a plain non-reactive value, composable cannot react on it.

`isActive` is get from `props`. Props in theory can change. But there will be no changes in this composable, when `isActive` prop is changed.

It should be a reactive variable `ref`. If we want to support both reactive `ref` and plain value, we can use it in composable via `unref`.

Reply:
So we should do something like this in th composable body?
```js
const isActiveRef = ref(isActive)
```

Reply:
I made it a ref now but I didn't convert it into a plain value in composable in order to watch its change.

Reply:
> So we should do something like this in th composable body?

No. This would just make a new ref, which initial value is based on initial prop value.

Example

```js
function useA(param) {
  // Not reactive, param is just a plain value
  return computed(() => param * 2)
}

function useB(param) {
  // Not reactive, paramRef is a new variable without relation with param
  const paramRef = ref(param)
  return computed(() => paramRef.value * 2)
}

function useC(paramRef) {
  // Reactive, but requires param to be ref (otherwise there is no .value)
  return computed(() => paramRef.value * 2)
}

function useD(param) {
  // Reactive, unversal - can be used with not reactive param
  // unref(param) = isRef(param) ? param.value : param
  return computed(() => unref(param) * 2)
}

export default defineComponent({
  props: {
    foo: String,
  },
  
  setup(props) {
    // Not reactive, a plain value (String) is passed
    useD(props.foo)
    
    // Reactive
    useD(toRef(props, 'foo'))
    // Reactive
    const { foo } = toRefs(props)
    useD(foo)
    // Reactive, Vue 3.2+
    useD(toRef(() => props.foo))
  },
})
```